apps/utils/start_server.py

```diff
@@ -0,0 +1,121 @@
+"""This script creates a new draft pull request in GitHub, which starts a new staging server.
+
+Usage:
+- If you use this script from the master branch on your local ETL repo, specify a --new-branch argument.
+  This will create a new local branch, which will be pushed to remote.
+- If you have already created a new local branch, run this script without specifying any argument.
+  The 'master' branch will be assumed as the base branch of the new draft pull request.
+- If you want the base branch to be different from 'master', specify it with the --base-branch argument.
+
+The resulting draft pull request will be new_branch -> base_branch.
+
+To create a GitHub token, go to GitHub settings. On the left, at the bottom, choose Developer settings.
+Then, choose "Personal access tokens" and "Tokens (classic)". Click on "Generate new token (classic)".
+Give it a name (e.g., "etl-work"), set an expiration time, and select the scope "repo".
+Then, click on Generate token. Copy the token and save it in your .env file as GITHUB_TOKEN.
+"""
+
+from typing import Optional
+
+import click
+import requests
+from git import Repo
+from rich_click.rich_command import RichCommand
+from structlog import get_logger
+
+from etl.config import GITHUB_TOKEN
+from etl.paths import BASE_DIR
+
+# Initialize logger.
+log = get_logger()
+
+# URL of the Github API, to be used to create a draft pull request in the ETL repos.
+GITHUB_API_URL = "https://api.github.com/repos/owid/etl/pulls"
+
+
+@click.command(name="start-server", cls=RichCommand, help=__doc__)
+@click.option(
+    "--new-branch",
+    type=str,
+    default=None,
+    help="Name of new branch to create (if not given, current branch will be used).",
+)
+@click.option(
+    "--base-branch", type=str, default=None, help="Name of base branch (if not given, 'master' will be used)."
+)
+def cli(new_branch: Optional[str] = None, base_branch: Optional[str] = None) -> None:
+    if not GITHUB_TOKEN:
+        log.error("Define GITHUB_TOKEN in your ETL .env file. Then run this tool again.")
```

Comment:
Can we either show full instructions how to create one or at least inform to run `etl d start-server --help` to get them?